Tasks/scss-targets.js

```diff
@@ -27,7 +24,7 @@ function compile(source, destination, compatibleBrowsers) {
         .pipe(cache('scss'))
         .pipe(plumber())
         .pipe(sourcemaps.init({ loadMaps: true }))
-        .pipe(sass({ linefeed: 'crlf' }))
+        .pipe(sass({ linefeed: 'crlf' }).on('error', sass.logError))
```

Comment:
The `.on('error', sass.logError))` seems to be duplicate now - see next line.

Reply:
Indeed, forgot it there. This also duplicated error messages in the output.